examples/src/keyrings/hierarchical_keyring.py

```diff
@@ -25,6 +55,7 @@
 
 from .example_branch_key_id_supplier import ExampleBranchKeyIdSupplier
 
+# TODO-MPL: Remove this as part of removing PYTHONPATH hacks.
```

Comment:
I assume this TODO is not yet TO BE DONE?

Reply:
This is still TODO. I have to:
* Update Dafny compiler to support outer-module
* Update Smithy-Dafny to reference outer-module
* Update MPL to use outer-module
I'm tracking that work separately.